runtime-common/yaml-cpp-0.7/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=yaml-cpp-0.7
+ABTYPE=cmake
```

Comment:
Weird placement, can you move this to above CMAKE_AFTER. Also, add a blank line after PKGDES.